### PR TITLE
ci: define file paths on events and fix gh release view args

### PR DESCRIPTION
**Pull request by fbozic**

### CODE REVIEW

File: .github/workflows/rust_binaries_release.yml [hunk 1]
```diff
@@ -4,12 +4,20 @@ on:
   push:
     branches:
       - master
-      - 'releases/**'
-      - 'hotfix/**'
```
Comment (miraclx) [inline .github/workflows/rust_binaries_release.yml:8]: can we restore this check for `pull_request`?

Comment (fbozic) [reply .github/workflows/rust_binaries_release.yml:8]: I can't filter the source branch like this, I could implement a step to check if PR + release branch at the begging and skip `build` and `release` steps conditionally.

Comment (miraclx) [reply .github/workflows/rust_binaries_release.yml:8]: yeah, that's what I meant.

```yml
    if: startsWith(github.head_ref, 'releases/')
```

or something like that, just to maintain the behaviour

Comment (fbozic) [reply .github/workflows/rust_binaries_release.yml:8]: done

**Review by miraclx — APPROVED** [review]

**Pull request merged**